org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,497 @@
+{
+    "app-id": "org.pitivi.Pitivi",
+    "branch": "1.0",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "finish-args": ["--command=pitivi",
+                    "--socket=x11",
+                    "--socket=pulseaudio",
+                    "--socket=wayland",
+                    "--socket=session-bus",
```

Comment:
Does it need access to entire session bus?

Reply:
Well, without it Pitivi doesn't start and we get:

`Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown`

but I must be missing something, couldn't figure out what :-)

Reply:
Probably just patch this to use the correct app-id: https://github.com/GNOME/pitivi/blob/ed321acfa2e84b67ce2ff7754ad135674b0aa74a/pitivi/application.py#L72